official/vision/beta/projects/yolo/modeling/backbones/Darknet.py

```diff
@@ -0,0 +1,356 @@
+import tensorflow as tf
+import tensorflow.keras as ks
+import collections
+
+from official.vision.beta.modeling.backbones import factory
```

Comment:
We need to add a unit test to make sure the models are created correctly. Can you add a test similar to the ones we have for resnet?

We want to make sure that the number of parameters here match the number of parameters for the official implementation of the paper.

Reply:
added the test and the doc string, to run the test call

```
python3 -m official.vision.beta.projects.yolo.modeling.tests.darknet_test
```